source/jsonnet/northern-ireland/individual/blocks/employment/employment_status_last_seven_days.jsonnet

```diff
@@ -63,17 +64,20 @@ local proxyTitle = {
   ],
 };
 
+local nonProxyDescription = 'If you have a job but have been off work on <em>furlough</em>, select “Temporarily away from work ill, on holiday or temporarily laid off”';
```

Comment:
We should follow the style guide and use a function for these descriptions.

Reply:
Done in #d1a750d